pages/docs/reference/type-safe-builders.md

```diff
@@ -2,20 +2,18 @@
 type: doc
 layout: reference
 category: "Syntax"
-title: "Type-Safe Groovy-Style Builders"
+title: "Type-Safe Builders"
 ---
 
 # Type-Safe Builders
 
-The concept of [builders](http://www.groovy-lang.org/dsls.html#_nodebuilder) is rather popular in the *Groovy* community.
-Builders allow for defining data in a semi-declarative way. Builders are good for [generating XML](http://www.groovy-lang.org/processing-xml.html#_creating_xml), 
-[laying out UI components](http://www.groovy-lang.org/swing.html), 
-[describing 3D scenes](http://www.artima.com/weblogs/viewpost.jsp?thread=296081) and more...
+By using well-named functions as builders in combination with extension lambdas it is possible to create static type safe builders in Kotlin. Type-safe builders are also known as domain specific languages (DSLs).
```

Comment:
Since DSLs in general are a much wider concept than the Kotlin type-safe builders, it could be better to say that the type-safe builders rather allow for creating Kotlin-based DSLs, with no implication that they are the same notion.